pages/common/dash.md

```diff
@@ -0,0 +1,20 @@
+# dash
+
+> Debian Almquist Shell, a modern, POSIX-compliant implementation of `sh` (not Bash-compatible).
+> More information: <https://wiki.archlinux.org/index.php/Dash>.
```

Comment:
```suggestion
> More information: <https://manned.org/dash>.
```
Is there a reason you didn't use the `manned.org` page?

Reply:
No there isn't, that in fact is a very good man page.